src/dotnet/APIView/APIViewWeb/Client/css/java.scss

```diff
@@ -1,47 +0,0 @@
-.icon-assembly {
```

Comment:
Why is this removed? This was added to support Language specific icons. are we supporting it in different way?

Reply:
Never mind. I see how you are handling it.